packages/react-charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -107,6 +91,35 @@ export class Arc extends React.Component<IArcProps, IArcState> {
     const yValue = point.yAxisCalloutData || point.data || 0;
     return point.callOutAccessibilityData?.ariaLabel || (legend ? `${legend}, ` : '') + `${yValue}.`;
   };
+
+  private _renderArcValue = (className: string) => {
+    const { arc, data, innerRadius, outerRadius, showValuesInPercent, totalValue, hideValues } = this.props;
+
+    if (hideValues || Math.abs(data!.endAngle - data!.startAngle) < Math.PI / 12) {
+      return null;
+    }
+
+    const [base, perp] = arc.centroid(data);
+    const hyp = Math.sqrt(base * base + perp * perp);
+    const textRadius = Math.max(innerRadius!, outerRadius!) + 2;
+    const angle = (data!.startAngle + data!.endAngle) / 2;
+    const arcValue = data!.value;
+
+    return (
+      <text
+        x={(hyp === 0 ? 0 : base / hyp) * textRadius}
+        y={(hyp === 0 ? 0 : perp / hyp) * textRadius}
+        textAnchor={angle > Math.PI ? 'end' : 'start'}
+        dominantBaseline={angle > Math.PI / 2 && angle < (3 * Math.PI) / 2 ? 'hanging' : 'auto'}
+        className={className}
+        aria-hidden={true}
```

Comment:
why aria-hidden. Does it not have accessibility text.

Reply:
The SVG text content is already accessible when cursor/focus is moved to the corresponding arc. So to prevent duplicate narration, the SVG text is made inaccessible

Reply:
Can you share the current narration.